### PR TITLE
chore: Make team area translatable

### LINKED ISSUE #5794: Teams Permission Page: Make all areas translatable
Seems like the teams permission page of events is not fully translatable yet.

Screenshot from 2020-11-27 21-02-49

### PR DESCRIPTION
**Pull request by SushantAA**
Fixes #5794 

#### Short description of what this resolves:
Adds translation to Teams permission page

#### Changes proposed in this pull request:
lang_change

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/events/view/team/permissions.ts [hunk 1]
```diff
@@ -155,65 +155,65 @@ export default class EventsViewTeamPermissions extends Route.extend({
           organizer : true
```
Comment (daretobedifferent18) [inline app/routes/events/view/team/permissions.ts:155]: why left this one?

File: app/routes/events/view/team/permissions.ts [hunk 2]
```diff
@@ -87,65 +87,65 @@ export default class EventsViewTeamPermissions extends Route.extend({
...
```
Comment (daretobedifferent18) [inline app/routes/events/view/team/permissions.ts:87]: also this one

File: app/routes/events/view/team/permissions.ts (hunk 1 above)
Comment (SushantAA) [reply app/routes/events/view/team/permissions.ts:155]: done

File: app/routes/events/view/team/permissions.ts (hunk 2 above)
Comment (SushantAA) [reply app/routes/events/view/team/permissions.ts:87]: done

Comment (SushantAA) [reply app/routes/events/view/team/permissions.ts:87]: can u pls give me the git command of upstream

**Review by MeghalBisht — COMMENTED** [review]
In your screenshot itself the translation is not happening! Please check again

**Comment by SushantAA** [comment]
It depends on language or full page is converted?

**Comment by SushantAA** [comment]
lang_change_c

**Comment by MeghalBisht** [comment]
The issue is to  make this area translatable as well

**Comment by SushantAA** [comment]
I think it depends on the language and the string. because the node module is not able to change the string some time. as I tried copying the same string it worked 
lang_depend

**Review by iamareebjamal — APPROVED** [review]
The issue is about making the area translatable, not to add the translations as well

**Comment by iamareebjamal** [comment]
@SushantAA The PR title is in reverse and not conforms to the guideline

**Comment by SushantAA** [comment]
@iamareebjamal  I will take care of that :sweat:

**Comment by iamareebjamal** [comment]
Yes, that's a valid review. Please be direct in your reviews like this. @SushantAA Please follow up

**Comment by SushantAA** [comment]
so i have to do  `<td class="left aligned">{{t-var permission.title}}</td>`

**Comment by iamareebjamal** [comment]
Yes

**Comment by SushantAA** [comment]
can u pls give me the upstream git command

**Comment by iamareebjamal** [comment]
What?

**Comment by SushantAA** [comment]
the 2 checkbox -- My branch is up-to-date with the Upstream development branch.
I don't know how to do upstream

**Comment by MeghalBisht** [comment]
https://www.atlassian.com/git/tutorials/git-forks-and-upstreams

**Comment by SushantAA** [comment]
I have done it and committed. Do I have to do another PR?

**Comment by MeghalBisht** [comment]
Yes

**Comment by SushantAA** [comment]
when I am doing Ustream my issue branch is not updated and all other updated

**Comment by MeghalBisht** [comment]
Please use a new branch for a new PR.

**Comment by SushantAA** [comment]
I have done  a new PR

**Pull request merged**